<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyerHomeWindowN</class>
 <widget class="QMainWindow" name="BuyerHomeWindowN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>622</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>622</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>771</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Список товаров</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>390</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Купить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Корзина</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>451</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="payButton">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>410</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Оплатить</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>601</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="totalPrice">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>350</y>
          <width>321</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>350</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Итого:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearCart">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>360</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Очистить корзину</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="backButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>410</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>60</y>
          <width>321</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите данные для оплаты:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cardNumber">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>140</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>16 цифр</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="monthYear">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>210</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>ММ/ГГ</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cvcCvv">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>210</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>3 цифры</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>110</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Номер карты</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>180</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Срок действия</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>180</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CVC / CVV</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="email">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>280</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>ivan_ivanov@mail.ru</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Адрес электронной почты</string>
        </property>
       </widget>
       <widget class="QPushButton" name="payButton_2">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>410</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Купить</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <zorder>nameLabel</zorder>
   <zorder>exitButton</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
